app/api/development/google-calendar/route.ts

```
@@ -0,0 +1,98 @@
+  if (!isDevelopment) {
+    return new NextResponse('Not available in production', { status: 403 })
+  }
+
+  try {
+    const session = await auth.api.getSession(request)
+    const userId = session?.user.id
+    const body = await request.json()
+
+    switch (body.action) {
+      case 'list_events':
+        const { timeMin, timeMax } = body
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)